src/main/resources/assets/garnished/lang/zh_cn.json

```diff
@@ -259,10 +259,10 @@
   "advancement.garnished.ina_buhg.description": "获得花生的一些变种",
   "advancement.garnished.walnut_brownie.title": "别发飙啦",
   "advancement.garnished.walnut_brownie.description": "获得核桃布朗尼",
-  "advancement.garnished.cashew_mixture.title": "Raise a Glass!", //
-  "advancement.garnished.cashew_mixture.description": "Obtain Bottled Fermented Cashew Mixture" //
-  "advancement.garnished.tophet_brew.title": "Dining in Hell", //
-  "advancement.garnished.tophet_brew.description": "Obtain a Tophet Brew" //
-  "advancement.garnished.aversion_effect.title": "Could've Been Worse", //
-  "advancement.garnished.aversion_effect.description": "Suffer from your new nut allergy" //
+  "advancement.garnished.cashew_mixture.title": "干杯！",
+  "advancement.garnished.cashew_mixture.description": "获得发酵腰果混合物瓶",
+  "advancement.garnished.tophet_brew.title": "宴于地狱",
+  "advancement.garnished.tophet_brew.description": "获得地狱煮",
+  "advancement.garnished.aversion_effect.title": "本还可以更糟",
+  "advancement.garnished.aversion_effect.description": "被新的坚果过敏折磨"
 }
```

Comment:
Advancement for consuming a Senile Sweet has also been added to lang files as of the most recent push